features/appmgt/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/apis/v1/asset_api_router.jag

```diff
@@ -1121,10 +1123,9 @@ require('/modules/publisher.js').exec(function(ctx) {
             print(msg);
 
         } catch (e) {
-            log.debug(e);
+            log.error(e);
+			log.error('The asset ' + id + ' could not be updated.');
```

Comment:
Followed this pattern (logging exception & message separately) because sometimes Jaggery version used in AppM does not format the stack trace of the exception properly if we logged message & exception at once `log.error(msg, e)`.